cli/command/engine/activate.go

```diff
@@ -90,6 +92,9 @@ func runActivate(cli command.Cli, options activateOptions) error {
 		if license, err = getLicenses(ctx, authConfig, cli, options); err != nil {
 			return err
 		}
+		if options.displayOnly {
```

Comment:
I guess the fix is here, but isn't it safer and simpler to just check if license is nil?
```go
if license, err = getLicenses(ctx, authConfig, cli, options); err != nil || license == nil {
    return err
}
```

Reply:
@dhiltgen ^

Reply:
@silvin-lubecki discussing with @dhiltgen out of band; let's merge this one as-is, and do a follow up with improvements